src/ops/ops.go

```diff
@@ -504,13 +507,123 @@ func (o *ops) RemovePV(pvName string) error {
 }
 
 func (o *ops) Wipefs(device string) error {
-	output, err := o.ExecPrivilegeCommand(o.logWriter, "wipefs", "-a", device)
+	_, err := o.ExecPrivilegeCommand(o.logWriter, "wipefs", "--all", "--force", device)
+
 	if err != nil {
-		o.log.Errorf("Failed to wipefs device %s, output %s, error %s", device, output, err)
+		_, err = o.ExecPrivilegeCommand(o.logWriter, "wipefs", "--all", device)
 	}
+
 	return err
 }
 
+func (o *ops) IsRaidDevice(device string) bool {
+	raidDevices, err := o.getRaidDevices()
+
+	if err != nil {
+		o.log.WithError(err).Errorf("Error occurred while trying to get list of raid devices - continue without cleaning")
+		return false
+	}
+
+	// The device itself or one of its partitions
+	expression, _ := regexp.Compile(device + "[\\d]*")
+
+	for _, raidArrayMembers := range raidDevices {
+		for _, raidMember := range raidArrayMembers {
+			if raidMember == device || expression.MatchString(raidMember) {
```

Comment:
`raidMember == device` is redundant because the regex already uses `*` rather than `+`

Reply:
sure,  but it's a simple and clear

Reply:
Fair enough. I personally find such redundancy confusing, it's makes the intent less clear - it made me question my understanding: "if the regex is capturing the device itself, why do we need this additional condition? what am I missing? what is it for?" and made me re-read a few times until I was sure it's just redundant

Reply:
It's about code styling but not critical for me. will update